### ISSUE TITLE
CPU output never appears at certain video length. 

### ISSUE THREAD

**Issue by gitihobo** [s0md3v/roop#359]
GPU fails at certain amount longer videos which is normal. But If I use only my cpu, the output never appears, shows no error, status doesn't change, it finishes processing part of the video but then never actually swaps or appears.

**Comment by henryruhs** [comment]
That is true, I actual fixed this on my local version of roop. Please reopen in case this is not fixed soon in the next branch.

**Comment by gitihobo** [comment]
can you apply your fix to this version as well?

**Comment by henryruhs** [comment]
You need to wait for next branch to be merged to main branch... not gonna happen before the weekend
